.github/workflows/tests-rs-package.yml

```diff
@@ -67,6 +71,37 @@ jobs:
       - name: Check formatting
         run: exit `cargo fmt --check --package=${{ inputs.package }} | wc -l`
 
+  unused_deps:
+    name: Unused dependencies
+    runs-on: ${{ fromJSON(inputs.lint-runner) }}
```

Comment:
ubuntu-22.04

Reply:
lint-runner is set to ubuntu-22.04 i test.yml

Reply:
But not for all packages. clippy for some of them very slow on free runner so we configure self-hosted runners for them

Reply:
set to ubuntu-22.04